lib/chatbot-api/chatbot-s3-buckets/index.ts

```diff
@@ -39,7 +40,31 @@ export class ChatBotS3Buckets extends Construct {
       ],
     });
 
+    const userFeedbackBucket = new s3.Bucket(this, "UserFeedbackBucket", {
```

Comment:
let's set the lifecycle rules here for simplicity and having them handled in one place (instead of object creation)

```
    const userFeedbackBucket = new s3.Bucket(this, 'UserFeedbackBucket', {
       ...
      lifecycleRules: [{
        transitions: [{
          storageClass: s3.StorageClass.INFREQUENT_ACCESS,
          transitionAfter: cdk.Duration.days(0), // Transition immediately after creation
        }],
      }],
    });
  }
 ```